### PR TITLE
Embedding support

### PR DESCRIPTION
**Pull request by ibaldin**
Added support to find nodes with matching selection of components. Will provide a matching PR for CF to use it.

### CODE REVIEW

**Comment by ibaldin** [comment]
Woops, this should not have the plugin code!!!

**Review by kthare10 — DISMISSED** [review]
Looks good!

File: fim/slivers/delegations.py [hunk 1]
```diff
@@ -440,6 +497,35 @@ def validate_pool(self):
         if self.get_pool_details() is None:
             raise PoolException(f"Pool {self.pool_id} does not have any resource details")
 
+    @staticmethod
+    def ispoolmention(self, d: Dict) -> bool:
```
Comment (kthare10) [inline fim/slivers/delegations.py:501]: Please remove self from variable list

Comment (ibaldin) [reply fim/slivers/delegations.py:501]: D'oh!

**Review by kthare10 — APPROVED** [review]

**Pull request merged**